src/main/java/frc/team2412/robot/util/autonomous/AutonomousChooser.java

```diff
@@ -165,13 +164,6 @@ public Pose2d getStartPose() {
         SHOOTER((subsystems) -> new ShooterTurretSetAngleCommand(subsystems.shooterSubsystem,
                 subsystems.shooterSubsystem.getTurretTestAngle()), "Shooter test",
                 Subsystems.SubsystemConstants.SHOOTER_ENABLED),
-        INTAKE_SHOOTER(
-                (subsystems) -> new ShootCommand(subsystems.indexSubsystem, subsystems.shooterSubsystem,
-                        subsystems.targetLocalizer),
-                "Intake and shoot",
-                Subsystems.SubsystemConstants.INTAKE_ENABLED &&
-                        Subsystems.SubsystemConstants.INDEX_ENABLED &&
-                        Subsystems.SubsystemConstants.SHOOTER_ENABLED),
```

Comment:
Not sure why this was removed, having an autonomous mode that intakes, indexes, and shoots is often useful for tuning shooter distance data points.